grunt/tasks/build.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(grunt) {
 
 	// Full distribution task
 	grunt.registerTask('dist', 'Build "dist." Contributors: do not commit "dist."',
-			['clean:dist', 'distcss', 'copy:fonts', 'copy:templates', 'distjs', 'commonjs', 'distzip']);
+			['clean:dist', 'distcss', 'copy:fonts', 'copy:templates', 'distjs', 'distzip']);
```

Comment:
It's used by CommonJS users of Fuel UX and anyone that pulls in FuelUX via NPM I would assume.